.github/CONTRIBUTING.md

```
@@ -6,7 +6,8 @@
 
 A guide to contributing to the WolfStar project<br/>
 
-[Discord Server] · [ESLint Setup][eslint] · [Development Guide](#running-typescript--gateway-based-wolfstar-project-bots-locally)
+[Discord Server] · [ESLint Setup][eslint] ·
+[Development Guide](#running-typescript--gateway-based-wolfstar-project-bots-locally)
```

markdownlint-cli2 (0.22.1)

[warning] Link fragments should be valid

(MD051, link-fragments)